python/sglang/srt/layers/quantization/w8a8_int8.py

```diff
@@ -34,16 +178,47 @@ class W8A8Int8Config(QuantizationConfig):
     - Activation: dynamic, per-token, symmetric
     """
 
-    def __init__(self):
-        pass
+    def __init__(self, quant_config: Dict[str, Any]):
```

Comment:
Here, after the code change, when creating `W8A8Int8Config`, `quant_config` is necessary. But, I search `W8A8Int8Config` in SGLang, no `quant_config` is provided. And when running `meituan/DeepSeek-R1-Channel-INT8`, it will encounter `TypeError: W8A8Int8Config.__init__() missing 1 required positional argument: 'quant_config'`.

Could you please fix this by passing `quant_config` or providing a default value? Thanks!

Reply:
Our colleague has drafted https://github.com/sgl-project/sglang/pull/7820 to fix this. Sorry for bothering.